docker-compose-production.yml

```diff
@@ -0,0 +1,56 @@
+# Use this compose file to run a local production-like stack
+#  `docker-compose --file docker-compose-production.yml` to use it up
+# Not great for development work
+version: '3'
+services:
+  app:
+    build: 
+      context: .
+      dockerfile: production.Dockerfile
+    volumes:
+       - ./app:/upaya/app
+       - ./certs:/upaya/certs
+       - ./config:/upaya/config
+       - ./db:/upaya/db
+       - ./keys:/upaya/keys
+       - ./lib:/upaya/lib
+       - ./log:/upaya/log
+       - ./pwned_passwords:/upaya/pwned_passwords
+       - ./vendor:/upaya/vendor
+    environment:
+      redis_url: "redis://redis:6379"
+      redis_throttle_url: "redis://redis:6379"
+      DATABASE_URL: "postgres://postgres@db"
+      DATABASE_CLEANER_ALLOW_REMOTE_DATABASE_URL: 'true'
+      # Set database timeouts to 30 seconds
+      database_timeout: '30000'
+      database_statement_timeout: '30000'
+      RACK_TIMEOUT_SERVICE_TIMEOUT: '30000'
+      DOCKER_DB_HOST: 'db'
+      DOCKER_DB_USER: 'postgres'
+      # '' == 1 thread for tests; performs better in a container
+      TEST_ENV_NUMBER: ''
+      SMTP_HOST: 'mailcatcher'
+    depends_on:
+      - db
+      - redis
+  #    - web
```

Comment:
Oh, one more Q that I forgot about, should this not be commented out?

Reply:
So... "web" is a placeholder where I was thinking we could put a NGINX container so you could test a prod-like system locally.  (For asset serving, for instance.)   I am not really tied to having a "production" compose file.  Maybe we should just kill it for now?

Reply:
I think it's OK to have for now, I was just wondering why the `depends` was commented out but the `web` image was still specified.

Reply:
I don't think it needs to block merge tho :)